libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -434,6 +441,8 @@ void AP_ADSB::handle_adsb_vehicle(const adsb_vehicle_t &vehicle)
     const uint16_t required_flags_position = ADSB_FLAGS_VALID_COORDS | ADSB_FLAGS_VALID_ALTITUDE;
     const bool detected_ourself = (out_state.cfg.ICAO_id != 0) && ((uint32_t)out_state.cfg.ICAO_id == vehicle.info.ICAO_address);
 
+    vehicle.last_update_ms = now - uint32_t(vehicle.info.tslc * 1000U);
```

Comment:
tslc is coming from external sources.  It should probably be validated before being used here...  this is a pre-existing issue so not a blocker.

Reply:
I'll just remove the brand new commits for this and keep the old behavior.